packages/evm/src/evm.ts

```diff
@@ -135,9 +138,21 @@ export interface EVMOpts {
   customPrecompiles?: CustomPrecompile[]
 
   /*
-   * The External Interface Factory, used to build an External Interface when this is necessary
+   * The StateManager which is used to update the trie
    */
-  eei: EEIInterface
+  stateManager: StateManagerInterface
+
+  /**
+   *
+   */
+  blockchain?: Blockchain
+
+  /**
+   * This optional flag should be set to `true` if no blockchain is provided
+   * This is used the warn users that if they do not provide a blockchain,
+   * the default blockchain will be used, which always returns the 0 hash if a block is
+   */
+  enableDefaultBlockchain?: boolean
```

Comment:
I don't think we need this option.

I somewhat understand the intention to "warn" users. I would argue that this is somewhat intuitive though (if an EVM has no connection to whatever blockchain, no information from a/the blockchain can be read). So I think if we have this prominently in the docs that really should be enough.

At the same time the option causes a lot of bloat - like in the tests.